guide.css

```diff
@@ -10,9 +10,35 @@ img{
 }
 
 .sectionTitle{
+	background-color: #FBEFD5;
+	border:none;
+	cursor:pointer;
 	text-align: center;
 	font-size: 1.5em;
 	border-bottom: 2px solid gray;
+	width:100%;
+}
+
+.sectionTitle:hover {
+	background-color: #ccc;
+}
+
+.sectionTitle:after {
+	content: "\2796"; /* Unicode character for "minus" sign (-) */
+	font-size: 13px;
+	color: white;
+	float: right;
+	margin-left: 5px;
+}
+
+/* Collapsible section content. Note: visible by default */
+.collapsible-content {
+	display: inline;
+	overflow: hidden;
+}
+
+.active:after {
+	content: '\02795'; /* Unicode character for "plus" sign (+) */
```

Comment:
Just insert the Unicode character directly, no need to encode it

Reply:
Done